doc/_static/custom_styles.css

```
@@ -136,14 +136,14 @@ div#driver-popover-item .driver-navigation-btns {
 div#driver-popover-item .tour-arrow {
     font-size: 2em;
     line-height: 0;
-    padding-bottom: 0.05em;
+    font-weight: bold;
```

```suggestion
    font-weight: bold;
    padding-bottom: 0.3em;
```

As can be seen in the preview, it needs padding to line up properly.